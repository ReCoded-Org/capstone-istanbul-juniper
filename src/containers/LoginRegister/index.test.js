import React from "react";
import LoginRegisterPage from "./index.jsx";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";

describe("LoginRegisterPage Component", () => {
  it("matches the tree", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <LoginRegisterPage />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
