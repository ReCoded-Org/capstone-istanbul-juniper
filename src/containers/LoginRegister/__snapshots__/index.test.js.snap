// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginRegisterPage Component matches the tree 1`] = `
Array [
  <div
    className="ant-spin-nested-loading"
  >
    <div
      className="ant-spin-container"
    >
      <form
        noValidate={true}
        onSubmit={[Function]}
      >
        <div
          className="loginContainer"
        >
          <div
            className="loginContainer__loginTitle"
          >
            login.login
          </div>
          
          <div
            className="loginContainer__loginDialog"
          >
            <div
              className="loginContainer__loginDialog__inputLabel"
            >
              login.email
            </div>
            <div
              className="loginContainer__loginDialog__input"
            >
              <input
                className="ant-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                type="email"
                value=""
              />
            </div>
            <div
              className="loginContainer__loginDialog__inputLabel"
            >
              login.password
            </div>
            <div
              className="loginContainer__loginDialog__input"
            >
              <input
                className="ant-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                type="password"
                value=""
              />
            </div>
            <div
              className="loginContainer__loginDialog__forgotPasswordContainer"
            >
              <a
                href="/#"
                onClick={[Function]}
              >
                login.forgotPassword
              </a>
            </div>
            <button
              className="loginContainer__loginDialog__submitButton"
              type="submit"
            >
              login.login
            </button>
            <div
              className="loginContainer__loginDialog__registerLabel"
            >
              login.new
            </div>
            <button
              className="loginContainer__loginDialog__registerButton"
              onClick={[Function]}
              type="button"
            >
              login.creat
            </button>
            <div
              className="loginContainer__loginDialog__loginOptionsTitle"
            >
              login.oryoucan
            </div>
            <button
              className="loginContainer__loginDialog__facebookLoginBtn"
              onClick={[Function]}
              type="button"
            >
              login.withFacebook
            </button>
            <button
              className="loginContainer__loginDialog__googleLoginBtn"
              onClick={[Function]}
              type="button"
            >
              login.withGoogle
            </button>
          </div>
        </div>
      </form>
    </div>
  </div>,
  <div
    className="LoginRegister__footerContainer"
  >
    <img
      alt="footerKids"
      className="LoginRegister__footerContainer__kidsImage"
      src="LoginKids.svg"
    />
  </div>,
]
`;
